app/routes/first-time/eligibility/claim/bank-account-details.js

```diff
@@ -23,6 +23,12 @@ module.exports = function (router) {
             .then(function () {
               return res.redirect(`/application-submitted/${req.params.reference}`)
             })
+            .catch(function (error) {
+              next(error)
+            })
```

Comment:
Should only need one .catch. Remove the inner catch.

There is no unit test to test for this branch being hit.